src/renderer/components/layout/sidebar-item.tsx

```diff
@@ -6,42 +6,49 @@ import { cssNames, prevDefault } from "../../utils";
 import { observer } from "mobx-react";
 import { NavLink } from "react-router-dom";
 import { Icon } from "../icon";
-import { TabLayoutRoute } from "./tab-layout";
 import { sidebarStorage } from "./sidebar-storage";
 import { isActiveRoute } from "../../navigation";
 
 interface SidebarItemProps {
-  id: string; // Used to save nav item collapse/expand state in local storage
+  id: string; // unique id, used in storage and integration tests
   url: string;
-  text: React.ReactNode | string;
   className?: string;
+  title: React.ReactNode;
```

Comment:
Changing this has nothing to do with fixing the bug.

Reply:
And what? I don't calculate changed lines (just re-sorted a bit for better look)

Reply:
You changed a prop field name, that doesn't have anything to do with this bug fix.

Reply:
Yes, it fits better imho + it's used currently only in `Sidebar` and `SidebarItem`. So it's a good time to change if we want it.